web_src/js/modules/fomantic.js

```diff
@@ -35,9 +35,10 @@ export function initGiteaFomantic() {
 
     const isIn = arg?.animation?.endsWith(' in');
     const isOut = arg?.animation?.endsWith(' out');
+    const isScale = arg?.animation?.includes('scale');
 
     let ret;
-    if (arg === 'show' || isIn) {
+    if (arg === 'show' || isIn || isScale) {
```

Comment:
Are you sure it is right? `scale` can be used to "show something" or "hide something"

https://fomantic-ui.com/modules/transition.html#scale

Reply:
> Are you sure it is right? `scale` can be used to "show something" or "hide something"
> 
> https://fomantic-ui.com/modules/transition.html#scale

Thanks for correcting the code. Thought I couldn't find another usage in `scale` except class="ui multiple search normal selection dropdown". In that particular case, `scale` is used to show the labels. 
In fact, I think we should just move this class. However, I failed to do that because I'm not familiar with the current CSS.